pkgs/applications/version-management/git-branchstack/default.nix

```diff
@@ -13,7 +13,7 @@ buildPythonApplication rec {
     hash = "sha256-gja93LOcVCQ6l+Cygvsm+3uomvxtvUl6t23GIb/tKyQ=";
   };
 
-  buildInputs = [
+  propagatedBuildInputs = [
```

Comment:
Now spelled `dependencies`:

```suggestion
  dependencies = [
```

Reply:
(for python packages).

Reply:
https://github.com/NixOS/nixpkgs/pull/348315